### PR TITLE
Chore: remove contants

### PR DESCRIPTION
**Pull request by blumendorf**

### CODE REVIEW

File: src/content/Paragraph/Paragraph.js [hunk 1]
```diff
@@ -24,14 +21,22 @@ export const Paragraph = ({ className, tag, style, size, ...props }) => {
...
+  tag: oneOf(['p', 'span']),
+  size: oneOf([
+    'xxSmall',
+    'xSmall',
+    'small',
+    'medium',
+    'large',
+    'xLarge',
+    'xxLarge',
+  ]),
```
Comment (blumendorf) [inline src/content/Paragraph/Paragraph.js]: I am wondering if we should instead allow to pass a type of `H500`, `H400`, `H300`, `H200`, `H100`, `Content`, `Description`, `Support`, or `Code` instead to match predefined fonts and get rid of custom sizes?!

File: src/content/List/ListItem.js [hunk 2]
```diff
@@ -2,19 +2,15 @@ import React from 'react';
...
-import { OLT_NAMESPACE } from '../../constants';
-
 const ListItem = ({ children, className, term, ...props }) => {
-  const base = `${OLT_NAMESPACE}List`;
-
   return (
     <>
       {term && (
-        <li {...props} className={classnames(`${base}-term`, className)}>
+        <li {...props} className={classnames(`olt-List-term`, className)}>
```
Comment (drinchev) [inline src/content/List/ListItem.js]: better written `import * as olt from '@lightelligence/styles'; ... classnames(olt.ListItem )` ...

File: src/content/Paragraph/Paragraph.js (hunk 1 above)
Comment (drinchev) [reply src/content/Paragraph/Paragraph.js]: I think we don't want to have type at all. It's a paragraph after all. It should have only one font size :)

Comment (blumendorf) [reply src/content/Paragraph/Paragraph.js]: ... `Content` it is.

**Pull request merged**